ftplugin/htmlhugo.vim

```diff
@@ -5,6 +5,7 @@ endif
 runtime! ftplugin/html.vim
 
 setlocal path+=layouts,resources,content,archetypes,static,data,layouts/_default,layouts/partials
+setlocal includeexpr=v:fname\.'.html'
```

Comment:
What about using `suffixesadd` instead?

```suggestion
setlocal suffixesadd=.html
```

Reply:
Oh, I wasn't aware of it, lol.  Much nicer!